client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -23,7 +25,7 @@ import { getDomainsWithForwards } from 'calypso/state/selectors/get-email-forwar
 import { fetchSiteDomains } from 'calypso/state/sites/domains/actions';
 import { getDomainsBySiteId, isRequestingSiteDomains } from 'calypso/state/sites/domains/selectors';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
-import type { Site } from 'calypso/reader/list-manage/types';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
```

Comment:
I think this could still be `import type` which has the very minor advantage that (I think) it will be stripped out by the TypeScript compiler.